index.js

```diff
@@ -931,7 +942,19 @@ export const Undefined = NullaryTypeWithUrl
   ([])
   (x => type (x) === 'Undefined');
 
-//# env :: Array Type
+//# config.checkTypes :: Boolean
+//.
+//. Run-time type checking is not free; one may wish to pay the performance
+//. cost during development but not in production. Setting `config.checkTypes`
+//. to `false` disables type checking, even for functions already defined.
+//.
+//. One may choose to use an environment variable to control type checking:
```

Comment:
To me it's already implied (by *may choose*) that the code that follows is just an example. :woman_shrugging:

I'm happy to make this change since you think clarification necessary.

Reply:
Ah, whatever :p